resources/assets/components/UserOverview/index.js

```diff
@@ -72,6 +72,9 @@ class UserOverview extends React.Component {
         filter: {
           id: ids.join(),
         },
+        //@TODO: when we update TransformsRequest.php to include limit=all,
+        // update this API request to use that instead of abritrary 1000.
+        limit: 1000,
```

Comment:
We [cap this limit at `100`](https://github.com/DoSomething/rogue/blob/db848e7114f4ead73c89d960e0b93d1445df3b1f/app/Http/Controllers/Traits/TransformsRequests.php#L116-L120), and would likely want to continue doing that since allowing more than that number of items in a response could lead to pretty unwieldy request size & performance issues.

Reply:
If we do need to get this number of items all at once, I'd recommend paginating the calls (into multiple smaller batches) & maybe looking into using GraphQL to be able to request only the subset of campaign fields we actually use for the UI (since as the number of things on a campaign grows over time, this will get less and less performant).

Reply:
@DFurnes ah thanks for linking to [this](https://github.com/DoSomething/rogue/blob/db848e7114f4ead73c89d960e0b93d1445df3b1f/app/Http/Controllers/Traits/TransformsRequests.php#L116-L120)! Is this cap at `100` the number of pages or the number of objects? 

The issue here is the API response only returns 20 campaigns in a page so only 20 of the campaigns on the user's page have the correct campaign title while the rest have the campaign number instead of the title.

Reply:
That's 100 items per response. If you need more than 100 per page, you should make multiple paginated calls to request all the items you need.

Reply:
ah gotcha - thanks! So for this quick fix, can is it ok to change this to be `limit: 100` and then create a new ticket to what you said in [this](https://github.com/DoSomething/rogue/pull/816#discussion_r244875162) comment?

Reply:
Yup, that sounds good to me. 👍

Reply:
cool this is updated and ready for review! The Pivotal Card is [here](https://www.pivotaltracker.com/n/projects/2019429/stories/162987212)!